### PR TITLE
Feature#41 adding and removing products from wishlist

### LINKED ISSUE #41: Adding and removing products from wishlist
## REQUIREMENTS / DESCRIPTION

> As a customer, I want to be able to add items to a wishlist to keep tracking of the products I want to buy later and remove them from the list if I no longer want to buy it or if I have already bought it.

### PR DESCRIPTION
**Pull request by HenrikhWolf**
# Description

What did you implemented/Why did you implemented this:

- Fixed some issues with Wishlist
- Now products can be added to the wishlist
- Now products can be removed from the wishlist
- You can use the heart in the corner of a Product Card to add or remove that product to/from the wishlist
- Products show if they are already in the wishlist (full heart) or not yet (Outlined heart)
- A confirmation message is shown when a product is added or removed to/from the wishlist
- This based on #41 

Examples:

- What?: I've added everything needed related to the functionalities of a wishlist.
- Why?: These changes complete the user experience of saving a product to buy later.

## Testing

- Since the project is just initialized you only need install the basic packages to visualize the application
```npm install```
- Then you can initialize with an android emulator or via web
```npm start```
- Login if you are registered
- Browse products and like them

 Example:

- I've added coverage for testing all new methods. I used Faker for a few random user emails and names.

## Screenshots

imagen
imagen
imagen
imagen
imagen

### CODE REVIEW

**Review by IvanGodinez21 — APPROVED** [review]

File: src/components/WishConfirm.jsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+export default class WishConfirm extends Component{
+    render() {
+        return (
+        <View style={ConfirmedWish.container}>
+        <Text style={ConfirmedWish.message}> Succesfully <Text style={ConfirmedWish.added}>added</Text> to your wishlist. </Text>
+        </View>
+        )
+    }
+}
```
Comment (YilKanda) [inline src/components/WishConfirm.jsx]: Add a blank line

File: src/components/WishRmConfirm.jsx [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+export default class WishRmConfirm extends Component{
+    render() {
+        return (
+        <View style={ConfirmedWish.container}>
+        <Text style={ConfirmedWish.message}> Succesfully <Text style={ConfirmedWish.removed}>removed</Text> from your wishlist. </Text>
+        </View>
+        )
+    }
+}
```
Comment (YilKanda) [inline src/components/WishRmConfirm.jsx]: Add a blank line

File: src/navigation/onWishlist.jsx [hunk 3]
```diff
@@ -79,23 +36,17 @@ export default class OnWishlist extends Component {
...
-            <Wishlist {...props} searching={this.state.searching} filtered={this.state.filtered} />
-          )}
-        </Stack.Screen>
+        <Stack.Screen name='Wishlist' component={WishList} />
+
       </Stack.Navigator>
     );
   }
-}
+}
```
Comment (YilKanda) [inline src/navigation/onWishlist.jsx]: Add a blank line

File: src/screens/Categories/WishList.jsx [hunk 4]
```diff
@@ -57,8 +61,7 @@ export default class Wishlist extends Component {
...
         />
-        {this.props.searching && <SearchDropdown dataSource={this.props.filtered} />}
-      </View>
-    );
-  }
-}
+      )}
+    </View>
+  );
+};
```
Comment (YilKanda) [inline src/screens/Categories/WishList.jsx]: Add a blank line

File: src/styles/WishlistScreen/Confirmed.jsx [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+    color: COLORS.GRAY,
+    height: 53,
+    backgroundColor: COLORS.WHITE,
+    marginHorizontal: 15,
+    borderRadius: 25,
+    shadowColor: COLORS.LIGHT_GRAY,
+  },
+  added:{color: COLORS.GREEN},
+  removed:{color: COLORS.RED},
+})
```
Comment (YilKanda) [inline src/styles/WishlistScreen/Confirmed.jsx]: Add a blank line

File: src/components/WishConfirm.jsx (hunk 1 above)
Comment (HenrikhWolf) [reply src/components/WishConfirm.jsx]: Done.

File: src/components/WishRmConfirm.jsx (hunk 2 above)
Comment (HenrikhWolf) [reply src/components/WishRmConfirm.jsx]: Done.

File: src/navigation/onWishlist.jsx (hunk 3 above)
Comment (HenrikhWolf) [reply src/navigation/onWishlist.jsx]: Done.

File: src/screens/Categories/WishList.jsx (hunk 4 above)
Comment (HenrikhWolf) [reply src/screens/Categories/WishList.jsx]: Done.

File: src/styles/WishlistScreen/Confirmed.jsx (hunk 5 above)
Comment (HenrikhWolf) [reply src/styles/WishlistScreen/Confirmed.jsx]: Done.

**Review by EliasCode79 — APPROVED** [review]

**Review by InGerardo777 — APPROVED** [review]
LGTM 🍆

File: src/components/WishRmConfirm.jsx (hunk 2 above)
Comment (YilKanda) [inline src/components/WishRmConfirm.jsx]: Check the indentation of your file.

Comment (HenrikhWolf) [reply src/components/WishRmConfirm.jsx]: Done.

**Review by yovanha — APPROVED** [review]

**Review by YilKanda — APPROVED** [review]
LGTM!

**Pull request merged**